bundles/org.openhab.binding.gardena/src/main/resources/OH-INF/thing/bridge.xml

```diff
@@ -4,39 +4,31 @@
 	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
 	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
 
-	<!-- Gardena Smart Home Account -->
+	<!-- Gardena smart system Account -->
 	<bridge-type id="account">
-		<label>Gardena Smart Home Account</label>
-		<description>The Gardena Smart Home Account</description>
+		<label>Gardena smart system Account</label>
```

Comment:
Words in labels should be capitalized (except prepositions and so on). See https://www.openhab.org/docs/developer/bindings/thing-xml.html#formatting-labels-and-descriptions